app/staging/domain/offender-manager.js

```diff
@@ -0,0 +1,12 @@
+class WorkloadOwner {
+  constructor (id, omKey, omForename, omSurname, omTypeId, omGradeCode) {
+    this.id = id
+    this.omKey = omKey
```

Comment:
AFAIK the omKey is a staging schema specific identifier which concatenates the OM id, the team id and LDU id. We should have all of this info in a normalized form so, so probably don't need this property.